limbo_console.gd

```diff
@@ -115,9 +125,44 @@ func _input(p_event: InputEvent) -> void:
 			get_viewport().set_input_as_handled()
 
 
+func _process(delta: float) -> void:
+	if _is_done_sliding: return
```

Comment:
This can be better handled by `set_process(false|true)`. Just disable the `_process` callback altogether if it's not animating. This will save a bit of performance by avoiding calls to it every frame.